frontend/src/pages/Author.jsx

```diff
@@ -82,20 +90,20 @@ class Author extends Component {
     }
 
     getFollowButton() {
-        let followbutton;
+        let button;
         if (this.state.isSelf) {
             return null;
         }
 
-        else if (!this.state.isFollowing && !this.state.isSelf) {
-            followbutton = <Button positive onClick={this.sendFollowRequest}><Icon name = "user plus" />Follow</Button>
+        if (!this.state.isFollowing) {
+            button = <Button positive onClick={this.sendFollowRequest}><Icon name = "user plus" />Follow</Button>
         }
 
-        else if (this.state.isFollowing && !this.state.isSelf) {
-            followbutton = <Button negative onClick={this.sendUnfollowRequest}><Icon name = "user times"/>Unfollow</Button>
+        if (this.state.isFollowing || this.state.isFriends) {
+            button = <Button negative onClick={this.sendUnfollowRequest}><Icon name = "user times"/>Unfollow</Button>
         }
 
-        return <div>{followbutton}</div>;
+        return <div>{button}</div>;
```

Comment:
Naming could be better, but I can't think of a better name uwu

Reply:
make followbutton great again